packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +694,151 @@ class Viewport implements IViewport {
     return true;
   }
 
+  /**
+   * Resets the pan offset.  To do this, the options values
```

Comment:
It says `resets the pan offset` but it should define what is actually the resetting the offset instead of how to do this. The how to do this should be part of the function (preferrably step by step guide of how we are doing it), and the function documentation should be what this function achieves. I expected this function to be a simple `this.panOffset = [0,0]` but it is not.